### DISCUSSION TITLE
Vaultwarden Logs for Fail2Ban

### DISCUSSION THREAD

**Discussion by savelle-anthony** [tteck/Proxmox#2573] [Help Needed]
Does anyone know where I can find / turn on logging for vaultwarden? I'd like to experiment with hardening my vaultwarden lxc that is reversed proxied with npm. The admin interface mentions these settings:
![image](https://github.com/tteck/Proxmox/assets/58756716/1f97881c-9af8-4d08-bb36-e81461ec48b5)

But in my _/opt/vaultwarden/data/config.json_ doesn't look like the config file for those read-only settings. Where can those environment variables be changed? Also does this seem like a reasonable thing to investigate in the first place, I have a pretty secure password with 2FA enabled.

**Comment by Gilly79** [comment]
Hi, has anyone already found a solution. I can't find a log file and no incorrect logins are noted in Systemctl. I don't need this for fail2ban, but for crowdsec.
